Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -209,8 +178,8 @@ - (void)encodeWithCoder:(NSCoder *)coder
 
     [coder encodeBool:self.suppressesIncrementalRendering forKey:@"suppressesIncrementalRendering"];
 
-    if (_applicationNameForUserAgent)
-        [coder encodeObject:self.applicationNameForUserAgent forKey:@"applicationNameForUserAgent"];
+    if (NSString *name = self.applicationNameForUserAgent)
```

Comment:
Isn't this a behavior change? Previously this was checking if `_applicationNameForUserAgent ` was `std::nullopt`, now it's checking if the NSString* is nil.

Reply:
It could be.  I'll keep the null and nullopt states distinct in this PR.